lib/govuk_index/presenters/expanded_links_presenter.rb

```diff
@@ -62,6 +66,12 @@ def content_ids(collection)
       values_from_collection(expanded_links_item(collection), "content_id")
     end
 
+    def slugs(type, path_prefix)
```

Comment:
I think this is a nice cleanup.

Reply:
Only thought would be to change to to match: https://github.com/alphagov/rummager/pull/1053/files#diff-506cd05cce4cbe451fda1d266331a7bcR156

```
.gusb(%r{^/....
```

That way it will only match the prefix. :)

Reply:
I've made it more consistent. 👍 What do you think?